lib/web_ui/dev/steps/compile_tests_step.dart

```diff
@@ -130,14 +130,20 @@ Future<void> copyCanvasKitFiles({bool useLocalCanvasKit = false}) async {
     environment.wasmReleaseOutDir.path,
     'canvaskit.wasm',
   ));
-  final bool builtLocalCanvasKit = localCanvasKitWasm.existsSync() && useLocalCanvasKit;
+  final bool builtLocalCanvasKit = localCanvasKitWasm.existsSync();
+  if (useLocalCanvasKit && !builtLocalCanvasKit) {
+    print('WARNING: Requested to use local CanvasKit but could not find the '
+        'built CanvasKit at ${localCanvasKitWasm.path}. Falling back to '
+        'CanvasKit from CIPD.');
```

Comment:
Let's throw. Otherwise, this can silently switch on CI or something and lead to surprises.

Reply:
Done.